recipes/BulkEval.py

```
@@ -448,15 +660,25 @@ def iter_eval_groups(df: "pd.DataFrame", array_columns: list[str] | None):
 def _run_language_model(model: str, prompt: str, result: TaskResult):
-    ret = json.loads(
-        run_language_model(
-            model=model,
-            prompt=prompt,
-            response_format_type="json_object",
-        )[0]
-    )
-    assert isinstance(ret, dict)
-    result.llm_output.update(ret)
+    entries = []
+    for entries in run_language_model(
```

Ruff (0.15.21)

[warning] Loop control variable `entries` not used within loop body

(B007)